bot/src/plugins/github_issues.cr

```diff
@@ -1,8 +1,21 @@
 require "framework/plugin"
+require "http/client"
+require "json"
 
 class GithubIssues
   include Framework::Plugin
 
+  class Issue
+    json_mapping({
+        html_url: String, title: String
+      })
```

Comment:
Well, while we're nitpicking I guess, indentation should be like
```cr
json_mapping({
  html_url: String,
  title: String
})
```

that is closing braces to the same level as the call and arguments two spaces indented towards the call. For two elements I'd find single line acceptable too I think, `json_mapping({html_url: String, title: String})`